### PR TITLE
[4.0] Installer Database should not use popovers

### PR DESCRIPTION
**Pull request by brianteeman**
PR for #24748

Use the new code for accessible "popovers" as with locked items and searchtools. Also added a tabindex to ensure that the message can be reached by keyboard navigation

### before
image

### after
image

### CODE REVIEW

File: administrator/components/com_installer/tmpl/database/default.php [hunk 1]
```diff
@@ -94,12 +93,14 @@
 											<td>
 												<?php echo $extension->type_translated; ?>
 											</td>
-											<td class="d-none d-md-table-cell">
-												<span class="badge badge-<?php echo count($item['results']['error']) ? 'danger' : ($item['errorsCount'] ? 'warning' : 'success'); ?> hasPopover"
-													data-content="<ul><li><?php echo implode('</li><li>', $item['errorsMessage']); ?></li></ul>"
-													data-original-title="<?php echo Text::plural('COM_INSTALLER_MSG_DATABASE_ERRORS', $item['errorsCount']); ?>">
+											<td class="d-none d-md-table-cell" >
```
Comment (Quy) [inline administrator/components/com_installer/tmpl/database/default.php]:
```suggestion
											<td class="d-none d-md-table-cell">
```

File: administrator/components/com_installer/tmpl/database/default.php [hunk 2]
```diff
@@ -94,12 +93,14 @@
...
+												<span class="badge badge-<?php echo count($item['results']['error']) ? 'danger' : ($item['errorsCount'] ? 'warning' : 'success'); ?>" tabindex="0">
 													<?php echo Text::plural('COM_INSTALLER_MSG_DATABASE_ERRORS', $item['errorsCount']); ?>
 												</span>
+												<div role="tooltip" id="<?php echo $extension->name; ?>">
```
Comment (Quy) [inline administrator/components/com_installer/tmpl/database/default.php]: Not valid id. No spaces allowed.
`<div role="tooltip" id="Joomla CMS">`

Comment (brianteeman) [reply administrator/components/com_installer/tmpl/database/default.php]: Ok. Wasn't sure. Will change it later

Comment (brianteeman) [reply administrator/components/com_installer/tmpl/database/default.php]: simplified

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on a7c3a8600ff4902dc837fd996718d2b8f5c2a482<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25207">issues.joomla.org/tracker/joomla-cms/25207</a>.</sub>

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 3b8f1912efa8d3a6da26ff9433447cbee423da18<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25207">issues.joomla.org/tracker/joomla-cms/25207</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25207">issues.joomla.org/tracker/joomla-cms/25207</a>.</sub>

**Comment by roland-d** [comment]
Thank you.

**Comment by brianteeman** [comment]
thanks

**Pull request merged**